lib/killbill_client/models/invoice.rb

```diff
@@ -94,13 +94,14 @@ def trigger_invoice(account_id, target_date, user = nil, reason = nil, comment =
           end
         end
 
-        def trigger_invoice_dry_run(account_id, target_date, upcoming_invoice_target_date, options = {})
+        def trigger_invoice_dry_run(account_id, target_date, upcoming_invoice_target_date, plugin_property = [], options = {})
```

Comment:
I think this method needs to be modified to accept parameters corresponding to `user`, `reason`, `comment` similar to [this method](https://github.com/killbill/killbill-client-ruby/blob/0c897c2006f5a8e22bbc98176e91200ff02253cd/lib/killbill_client/models/admin.rb#L24).